### DISCUSSION TITLE
how to not use `remix-flat-routes` package?

### DISCUSSION THREAD

**Discussion by SomiDivian** [epicweb-dev/epic-stack#34] [Q&A]
Hey everyone

I've being trying to use this stack with the original `v2_routeConvention: true` , but it doesn't work

some error sometimes show up in the network tab 
```cli
**Provisional headers warning**
Provisional headers are shown...
```

**Comment by kentcdodds** [comment]
You'll need to update the `remix.config.js` to remove the flat routes. Then you'll need to move files around a fair bit. We're not planning on adding documentation for how to change the opinions like this.
